roles/openshift_metering/tasks/install.yaml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-metering-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Ensures {{ mktemp.stdout }}/metering-manifests directories exists
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+  - "{{ mktemp.stdout }}/metering-manifests/operator"
+
+- name: Copy operator files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/metering-manifests/operator/{{ item | basename }}"
+  with_fileglob:
+  - "files/operator/*.yaml"
+
+- name: Update Metering Helm Operator deployment image
+  yedit:
+    src: "{{ mktemp.stdout }}/metering-manifests/operator/metering-helm-operator-deployment.yaml"
```

Comment:
This deployment YAML doesn't seem to be used anywhere, it should be created with `oc_obj` too:

```
- name: Apply Controller Deployment manifest
  oc_obj:
    state: present
    kind: Deployment
    name: "metering-helm-operator"
    namespace: "{{ __openshift_metering_namespace }}"
    files:
    - "{{ mktemp.stdout }}/metering-manifests/operator/metering-helm-operator-deployment.yaml"
  run_once: true
```